web/client-api/src/main/java/io/deephaven/web/client/api/barrage/BarrageUtils.java

```diff
@@ -353,10 +342,8 @@ private static BitSet readValidityBufferAsBitset(ByteBuffer data, int size, Buff
             // these buffers are optional (and empty) if the column is empty, or if it has primitives and we've allowed DH nulls
             return new BitSet(0);
         }
-        int startPosition = data.position();
-        data.position((int) (data.position() + buffer.offset().toFloat64()));
+        data.position((int) buffer.offset().toFloat64());
```

Comment:
this is a permanent side effect. now; is that ok? I was worried, ish, about that.

Reply:
Since all offsets are absolute, any time we read an array out we have to move the position anyway, and any time we read a value we specify the position in the method call.

Note that i also removed all the slice() calls, since we are never worried about relative positions, this isn't as helpful.